IssueTriage.md

```diff
@@ -0,0 +1,104 @@
+# Triage Guidance for WebNN Repo
+
+TODO: Consider removing these, or fit these into the documentation below.
+
+- **dependencies** - unused?
+- **tag** - redundant with **tag-needs-resolution** and **tag-tracker** ?
+- **enhancement** - unclear if this is about the spec text, the proposed API, etc. Prefer **feature request** etc.
+- **help wanted** - unclear what help is needed; use **needs PR** or **question** instead?
+
+
+## Label Usage
+
+Labels are used for:
+
+- Understanding the status of a specific issue and next steps to resolve it.
+- Understanding the scope of work remaining on broad efforts (e.g. aligning with best practices, fixing normative issues, etc)
+- Identifying areas to contribute.
+
+The working group chairs and spec editors should regularly review bugs and ensure that labels are accurate, and ensure that issues are getting appropriate attention; for example, scheduling discussion of new feature requests, discussion to resolve outstanding questions, and drawing attention to issues that are ready for a contributor to author a PR.
+
+
+## Types
+
+Every issue should have one of these issue types, and only rarely more than one.
+
+- **bug** - a flaw in the specification that will require a normative fix; for example, an algorithm computes an output incorrectly
+- **conventions-integration** - where the spec does not conform to specification best practices from Web IDL, Bikeshed, Infra, etc.
+- **use case** - a new use case for the API that should be documented or considered; may spawn other issues
+- **process** - a meta issue about how the specification is evolved; for example, a discussion of issue labels
+- **testing** - discussion of test coverage
+- **feature request** - _PROPOSED_ - suggestion for an addition to the proposed API
+- **interop** - _PROPOSED_ - discussion of differences in behavior across implementations
+- **samples** - _PROPOSED_ - suggestions for additional or improved samples
+
+
+## Spec Impact
+
+These broad categories describe the projected impact on the specification and implementation of an issue.
+
+- **Editorial** - spec text could be improved, but does not imply normative behavior changes to the proposed API
+- **Normative** - _PROPOSED_ - resolving the issue would require behavior changes, likely affecting implementations and tests
+
+TODO: I'm not convinced of the utility of these; maybe drop them?
+
+TODO: Make these lowercase for consistency?
+
+## Issue Scope
+
+Every issue should have at least one of these issue types, but occasionally multiple.
+
+- **operation set** - discussions about the overall operator coverage of WebNN; examples include alignment with other published operator sets, use cases that require multiple new operators, compatibility with implementations, etc.
+- **operator specific** - _PROPOSED_ - issues regarding the specification of a single operator or small number of operators
```

Comment:
```suggestion
- **operation set** - discussions about the overall operation coverage of WebNN; examples include alignment with other published operation sets (op set), use cases that require multiple new operations, compatibility with implementations, etc.
- **operation specific** - _PROPOSED_ - issues regarding the specification of a single operation or small number of operations
```

Are we making someone unhappy if we standardize on _operation_? :)

I note the spec needs some regexp action for this because we use operation and operator interchangeably.

Reply:
It looks like most frameworks (TF, ONNX, PyTorch) use "operator" more frequently in this context, e.g. "operator set", "operators", "operator compatibility", "operator coverage".  (thanks @a-sully) 

_Operators_ perform _operations_, so the phrase _"a platform operator for the operation"_ correctly uses both. 

For the labels, I think "operator" would align with more common usage. But I'm still new to all this!

Reply:
Let’s go with the most popular name for the labels. We can adjust these later as needed.